src/cli.ts

```diff
@@ -165,29 +169,20 @@ export class CLI {
 		}
 	}
 
-	public execute<TData extends string | Record<string, any> | void>(
-		command: string[],
-		{
-			args = [],
-			flags = {},
-			stdin,
-			json = true,
-		}: {
-			args?: (string | null | FieldAssignment)[];
-			flags?: Flags;
-			stdin?: string | Record<string, any>;
-			json?: boolean;
-		} = {},
-	): TData {
-		let input: NodeJS.ArrayBufferView;
-		command = command.map((part) => sanitizeInput(part));
+	private createParts(
+		subCommand: string[],
+		args: Arg[],
```

Comment:
This makes me think we should change the type a bit. Currently this allows a type of `null[]` which doesn't make a lot of sense for a few reasons.

1. I tend to lean to `undefined` over `null` in all cases because it reduces a lot of unpredictability in certain situations, and behaves more consistently.
2. I think we should remove `null` from the `Arg` type alias, and instead type this parameter as `Arg[] | undefined`

Reply:
Can you give an example of an instance where we'd need to be able to pass an argument value like `[null]`?

Would it make sense to just remove the `| null` from the `type Arg`?

Reply:
Hehe, I guess I can't. It's possible I was doing something funky while op-vscode was being developed that I wanted this for, but I can't find it anymore. I've removed this option :D

Reply:
Perfect 😅